sph/post_processing.py

```diff
@@ -85,3 +86,30 @@ def create_movie_from_vtk(vtk_output_dir: str,
         frames = sorted(frames)
         clip = moviepy_io.ImageSequenceClip(frames, fps=fps)
         clip.write_videofile(movie_path)
+
+
+def render(source_dir,
```

Comment:
swap the functions:
`render` calls `create_movie_from_vtk` so it is natural that render comes first.

Reply:
I would assume the reverse, since I need the `create_movie_from_vtk` to create `render`. But I guess for persons checking the code, the perception changes.